src/app/public/modules/label/label.component.ts

```diff
@@ -11,17 +15,18 @@ import {
 export class SkyLabelComponent {
 
   /**
-   * The type of label to display. Possible values are `info`, `success`, `warning` and `danger`.
+   * The type of label to display.
    * @required
    */
   @Input()
-  public set labelType(value: string) {
+  public set labelType(value: SkyLabelType) {
```

Comment:
![I see what you did there](https://media.giphy.com/media/ANbD1CCdA3iI8/giphy.gif)

Reply:
HA!